src/ert/ensemble_evaluator/_builder/_legacy.py

```diff
@@ -138,61 +143,24 @@ async def send_timeout_message() -> None:
 
         return on_timeout, send_timeout_future
 
-    def evaluate(self, config: "EvaluatorServerConfig") -> None:
-        if not config:
-            raise ValueError("no config for evaluator")
-        self._config = config
-        get_event_loop().run_until_complete(
-            wait_for_evaluator(
-                base_url=self._config.url,
-                token=self._config.token,
-                cert=self._config.cert,
-            )
-        )
-
-        threading.Thread(target=self._evaluate, name="LegacyEnsemble").start()
-
-    def _evaluate(self) -> None:
-        """
-        This method is executed on a separate thread, i.e. in parallel
-        with other threads. Its sole purpose is to execute and wait for
-        a coroutine
-        """
-        # Get a fresh eventloop
-        asyncio.set_event_loop(asyncio.new_event_loop())
-
-        if self._config is None:
-            raise ValueError("no config")
-
-        # The cloudevent_unary_send only accepts a cloud event, but in order to
-        # send cloud events over the network, we need token, URI and cert. These are
-        # not known until evaluate() is called and _config is set. So in a hacky
-        # fashion, we create the partialmethod (bound partial) here, after evaluate().
-        # Note that this is the "sync" version of evaluate(), and that the "async"
-        # version uses a different cloudevent_unary_send.
-        self.__class__._ce_unary_send = partialmethod(  # type: ignore
-            self.__class__.send_cloudevent,
-            self._config.dispatch_uri,
-            token=self._config.token,
-            cert=self._config.cert,
-        )
-        try:
-            get_event_loop().run_until_complete(
-                self._evaluate_inner(
-                    cloudevent_unary_send=self._ce_unary_send  # type: ignore
-                )
-            )
-        finally:
-            get_event_loop().close()
-
     async def evaluate_async(
```

Comment:
Perhaps we can now just rename this to `evaluate`, as it is the only function named `evaluate`.